clang/test/dpct/cmake_migration/case_001/expected.txt

```diff
@@ -35,6 +35,4 @@ get_filename_component(bin_path_of_dpct ${dpct_bin_path} DIRECTORY)
 set(dpct_cmake_file_path "${bin_path_of_dpct}/../cmake/dpct.cmake")
 include(${dpct_cmake_file_path})
 
-target_link_libraries(foo3 PUBLIC  ${TCNN_LIBRARIES} fmt)
```

Comment:
Removing here because target_link_libraries is handled with DPCT helper function

Reply:
Pls submit a separate PR to do that.